project/V.scala

```diff
@@ -97,7 +97,7 @@ object V {
     Seq(scala3, "3.2.1", "3.2.0", "3.1.3") ++ scala3RC.toSeq
```

Comment:
Don't we also need to bump `scala3RC` up above?

Reply:
uch, something weird happened there :sweat_smile:  Thanks for noticing!